vars/buildPipeline.groovy

```diff
@@ -51,6 +51,24 @@ def call(String jobName, CloneStep cloneStep, java.util.ArrayList<BuildStep> bui
           }
         }
       }
+      stage("Build Project Images") {
```

Comment:
This should likely have a "when" clause that only runs if there are any imageBuildSteps 
Also, should create a configmap that will contains the rpm repo URL

Reply:
// TODO :)

Reply:
still working